Detectors/FIT/FT0/calibration/testWorkflow/FT0TFProcessor-Workflow.cxx

```diff
@@ -0,0 +1,78 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+
+#include <Framework/ConfigContext.h>
+#include "Framework/DeviceSpec.h"
+#include "Framework/WorkflowSpec.h"
+#include "Framework/CompletionPolicyHelpers.h"
+#include "Framework/Task.h"
+#include "DataFormatsFT0/ChannelData.h"
+#include "DataFormatsFT0/Digit.h"
+#include "FT0Calibration/FT0CalibrationInfoObject.h"
+
+using namespace o2::framework;
+
+namespace o2::calibration::fit
+{
+
+class FT0TFProcessor final : public o2::framework::Task
+{
+
+ public:
+  void run(o2::framework::ProcessingContext& pc) final
+  {
+    auto channels = pc.inputs().get<gsl::span<o2::ft0::ChannelData>>("channels");
+    auto digits = pc.inputs().get<gsl::span<o2::ft0::Digit>>("digits");
+    auto& calib_data = pc.outputs().make<std::vector<o2::calibration::fit::FT0CalibrationInfoObject>>(o2::framework::OutputRef{"calib", 0});
+    calib_data.reserve(channels.size());
+
+    for (const auto& channel : channels) {
+      calib_data.emplace_back(channel.getChannelID(), channel.getTime());
+    }
+  }
+};
+
+}
+
+void customize(std::vector<o2::framework::CompletionPolicy>& policies)
+{
+  policies.emplace_back(CompletionPolicyHelpers::processWhenAny());
+}
```

Comment:
Because I do not want to resend whole digits object.
Default policy is consume -> resend receceived objects, but process won't do it. (Or maybe again I can be wrong).

Reply:
I have no particular experience with completion policies, but 1st of all, I think it should be applied to particular device and I don't understand what do you mean by resending? As far as I understand, the "consume" means that already received inputs will not be visible in the same device with the inputs arriving later. Perhaps @ktf can confirm.

In any case: for this device I don't see what you can process if only 1 of 2 inputs you request is there (not to say that this will never happen since they are sent by the same device together)...

Reply:
Yup, with some further investigation I do not have to change CompletionPolicy.

Reply:
so, if you don't need it, will you remove it?